recipes/loopdesigner/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: loopdesigner
+  version: 0.1.1
+
+source:
+  url: https://git.ligo.org/gabriele-vajente/loopdesigner/-/archive/0.1.1/loopdesigner-0.1.1.tar.gz
+  sha256: 92ef172d2a990af29576f968e59388e5b8e1de6f2c4f17fa0b9eea589d336f34
+
+build:
+  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+  skip: true  # [win]
```

Comment:
I believe this one can be noarch.

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.